src/Client.js

```diff
@@ -610,6 +610,43 @@ class Client extends EventEmitter {
         }, chatId);
     }
 
+    /**
+     * Enables and returns the pin state of the Chat
+     * @returns {boolean}
+     */
+    async pinChat(chatId) {
+        return await this.pupPage.evaluate(async chatId => {
+            let chat = await window.Store.Chat.get(chatId);
```

Comment:
```suggestion
            let chat = window.Store.Chat.get(chatId);
```

No need for `await` here. Getting a chat does not return a promise.